content/en/docs/contribute/style/style-guide.md

```diff
@@ -146,6 +146,33 @@ Set the value of the `replicas` field in the configuration file. | Set the value
 The value of the `exec` field is an ExecAction object. | The value of the "exec" field is an ExecAction object.
 {{< /table >}}
 
+### Use code style for Kubernetes command tool and component names
+
+{{< table caption = "Do and Don't - Use code style for Kubernetes command tool and component names" >}}
+Do | Don't
+:--| :-----
+The `kubelet` preserves node stability. | The kubelet preserves node stability.
```

Comment:
Sometimes it seems helpful to use a glossary tooltip so that readers easily know what components are.

Reply:
I favor writing `kubectl` in code style, because people type it. Same for other CLI tools. If the option were there, I would write `kube-apiserver` in monospace but not styled as code.

Reply:
@sftim, glossary tooltips are helpful, but they can be disruptive if overused on a page.
It is my understanding that using backticks is supposed to render text in monospace.
I can change the component example to use plain text, though, I'd like a different font or font size for k8s component names.